calyx-ir/src/control.rs

```diff
@@ -477,6 +479,11 @@ impl Control {
         })
     }
 
+    /// Convience constructor for a static control
+    pub fn static_control(sc: StaticControl) -> Self {
+        Control::Static(sc)
+    }
```

Comment:
Probably worthwhile to just implement `From<StaticControl>` for `Control` and
use that instead